groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group10/PlatinumGroupChain.groovy

```diff
@@ -9,117 +9,920 @@ MACERATOR = recipemap('macerator')
-    .fluidInputs(fluid('ammonia_solution') * 1000)
-    .fluidInputs(fluid('alluvial_divalent_palladium_solution') * 7000) //(H2PdCl4)(ZnCl2)(HCl)4(H2O)14
-    .chancedOutput(metaitem('dustDiamminedichloropalladium') * 11, 3750, 0)
-    .fluidOutputs(fluid('alluvial_palladium_mother_liquor') * 8000) //(NH4Cl)2(ZnCl2)(HCl)4(H2O)16
-    .duration(60)
-    .EUt(Globals.voltAmps[2])
-    .buildAndRegister()
+        .fluidInputs(fluid('ammonia_solution') * 2000)
+        .fluidInputs(fluid('alluvial_divalent_palladium_solution') * 14000) //(H2PdCl4)(ZnCl2)(HCl)4(H2O)14
+        .chancedOutput(metaitem('dustDiamminedichloropalladium') * 11, 7500, 0)
+        .fluidOutputs(fluid('alluvial_palladium_mother_liquor') * 16000) //(NH4Cl)2(ZnCl2)(HCl)4(H2O)16
+        .duration(120)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
 
 REACTION_FURNACE.recipeBuilder()
-    .inputs(ore('dustDiamminedichloropalladium') * 11)
-    .fluidInputs(fluid('hydrogen') * 4000)
-    .outputs(metaitem('dustPalladium'))
-    .fluidOutputs(fluid('ammonia') * 2000)
-    .fluidOutputs(fluid('hydrogen_chloride') * 2000)
-    .duration(240)
-    .EUt(Globals.voltAmps[2])
-    .buildAndRegister()
+        .inputs(ore('dustDiamminedichloropalladium') * 11)
+        .fluidInputs(fluid('hydrogen') * 4000)
+        .outputs(metaitem('dustPalladium'))
+        .fluidOutputs(fluid('ammonia') * 2000)
+        .fluidOutputs(fluid('hydrogen_chloride') * 2000)
+        .duration(240)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
 
 DISTILLATION_TOWER.recipeBuilder()
-    .fluidInputs(fluid('alluvial_palladium_mother_liquor') * 16000)
-    .chancedOutput(metaitem('dustZincChloride') * 3, 9500, 0)
-    .fluidOutputs(fluid('ammonium_chloride_solution') * 2000)
-    .fluidOutputs(fluid('water') * 14000)
-    .fluidOutputs(fluid('hydrogen_chloride') * 4000)
-    .duration(240)
-    .EUt(Globals.voltAmps[2])
-    .buildAndRegister()
+        .fluidInputs(fluid('alluvial_palladium_mother_liquor') * 16000)
+        .chancedOutput(metaitem('dustZincChloride') * 3, 9500, 0)
+        .fluidOutputs(fluid('ammonium_chloride_solution') * 2000)
+        .fluidOutputs(fluid('water') * 14000)
+        .fluidOutputs(fluid('hydrogen_chloride') * 4000)
+        .duration(240)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
 
 //MODERN SEPARATION PROCESSES
+GRAVITY_SEPARATOR.recipeBuilder() 
+        .inputs(ore('dustPentlandite'))
+        .outputs(metaitem('dustSiftedPentlandite'))
+        .chancedOutput(metaitem('dustUltramaficTailings'), 2500, 0)
+        .EUt(Globals.voltAmps[1])
+        .duration(40)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustSiftedPentlandite') * 8)
+        .fluidInputs(fluid('distilled_water') * 2000)
+        .fluidOutputs(fluid('impure_pentlandite_slurry') * 2000)
+        .EUt(Globals.voltAmps[3])
+        .duration(80)
+        .buildAndRegister()
+
+FF.recipeBuilder()
```

Comment:
this is an extremely bizarre set of flotation reagents. industry uses sodium isobutyl xanthate as a collector, copper sulfate as an activator, and cresol as a frother.